src/commands/accesslogs.js

```diff
@@ -1,42 +1,93 @@
 'use strict';
 
-const { getAccessLogsFromWarp10InBatches, getContinuousAccessLogsFromWarp10 } = require('@clevercloud/client/cjs/access-logs.js');
-const { getWarp10AccessLogsToken } = require('@clevercloud/client/cjs/api/v2/warp-10.js');
-const { ONE_HOUR_MICROS, ONE_SECOND_MICROS, toMicroTimestamp } = require('@clevercloud/client/cjs/utils/date.js');
-
 const Addon = require('../models/addon.js');
 const AppConfig = require('../models/app_configuration.js');
 const Logger = require('../logger.js');
-const { getFormatter } = require('../models/accesslogs.js');
-const { sendToApi, sendToWarp10 } = require('../models/send-to-api.js');
+const { getHostAndTokens } = require('../models/send-to-api.js');
+const { ApplicationAccessLogStream } = require('@clevercloud/client/cjs/streams/access-logs.js');
+const { JsonArray } = require('../models/json-array.js');
+const colors = require('colors/safe');
 
-const CONTINUOUS_DELAY = ONE_SECOND_MICROS * 5;
+// 2000 logs per 100ms maximum
+const THROTTLE_ELEMENTS = 2000;
+const THROTTLE_PER_IN_MILLISECONDS = 100;
 
 async function accessLogs (params) {
-  const { alias, format, before, after, addon: addonId, follow } = params.options;
+  const { apiHost, tokens } = await getHostAndTokens();
+  const { alias, format, before, after, addon: addonId } = params.options;
+  const { ownerId, appId } = await getIds(addonId, alias);
```

Comment:
issue: I think we should detect early if the addonId is used and throw an error saying support for add-ons is not available for now.